### PR TITLE
Add cluster url param and rename tds to testnet

### PR DESCRIPTION
**Pull request by jstarry**

### CODE REVIEW

File: src/providers/network.tsx [hunk 1]
```diff
@@ -10,14 +10,14 @@ export enum NetworkStatus {
 
 export enum Network {
```
Comment (mvines) [inline src/providers/network.tsx]:
```suggestion
export enum Cluster {
```

File: src/providers/network.tsx [hunk 2]
```diff
@@ -10,14 +10,14 @@ export enum NetworkStatus {
...
   MainnetBeta,
-  TdS,
+  Testnet,
   Devnet,
   Custom
 }
 
 export const NETWORKS = [
```
Comment (mvines) [inline src/providers/network.tsx]:
```suggestion
export const CLUSTER = [
```

File: src/providers/network.tsx [hunk 3]
```diff
@@ -10,14 +10,14 @@ export enum NetworkStatus {
...
   Network.MainnetBeta,
```
Comment (mvines) [inline src/providers/network.tsx]:
```suggestion
  Cluster.MainnetBeta,
```
etc...

**Comment by jstarry** [comment]
heh, been putting that off. I'll follow up with a full rename PR

**Comment by jstarry** [comment]
CI is backed up, added the rename to this PR

**Pull request merged**